lab_6_pipeline/pipeline.py

```diff
@@ -67,11 +124,16 @@ def __init__(
             corpus_manager (CorpusManager): CorpusManager instance
             analyzer (LibraryWrapper | None): Analyzer instance
         """
+        self._corpus = corpus_manager
+        self.analyzer = analyzer
 
     def run(self) -> None:
         """
         Perform basic preprocessing and write processed text to files.
         """
+        articles = self._corpus.get_articles()
+        for article in articles.values():
+            io.to_cleaned(article)
```

Comment:
There are only few students which using only this method without any preprocessing. I am glad to see this piece code!